c++/ssc_kosaraju_algo.cpp

```diff
@@ -0,0 +1,136 @@
+//Somos insignificantes. Por mais que você programe sua vida, a qualquer momento tudo pode mudar.
```

Comment:
#1 , just curious, why would someone write a quote  in Portuguese here ? 👀

Reply:
Sorry that was part of my template although it serves no purpose in implementation .
It's just that I like this quote and used in my template and forgot to edit it out

Reply:
also, you were supposed to push it from a branch other than main.

Reply:
Sure 
I'll take care of it from next time 👍